api/account/account_api/endpoints/device.py

```diff
@@ -244,81 +241,43 @@ def _determine_disconnect_duration(last_contact_age: timedelta) -> str:
     def post(self):
         """Handle a HTTP POST request."""
         self._authenticate()
-        device = self._validate_request()
-        self._pair_device(device)
+        self._validate_request()
```

Comment:
This is clearly not my area of expertise. It kind of surprises me that the endpoint has a single request property. Rather than the request itself being a class or passing in a request object to the method or something.

Reply:
This was done to get around naming convention issues.  In the Typescript code, camel case is used.  In Python code, snake case is used.  In order to correctly call the database access methods, the field names need to be in snake case.